lib/evmone/instructions_calls.cpp

```diff
@@ -324,6 +321,32 @@ Result eofcreate(
     if (!check_memory(gas_left, state.memory, input_offset_u256, input_size_u256))
         return {EVMC_OUT_OF_GAS, gas_left};
 
+    bytes_view initcontainer;
+    if constexpr (Op == OP_EOFCREATE)
+    {
+        const auto initcontainer_index = uint8_t{pos[1]};
+        pos += 2;
+        const auto& container = state.original_code;
+        const auto eof_header = read_valid_eof1_header(state.original_code);
```

Comment:
You're right, by this point we've already read it, and it's accessible, but only in `state.analysis.baseline`.

Maybe it's better to move it `ExecutionState`, otherwise it's like an easy way to crash Advanced.

Reply:
Another idea is to put array of `bytes_view`s to all containers into `ExecutionState`, similar to what we do now with data section.